ntrans_gui.py

```diff
@@ -16,6 +17,7 @@ class NTransMainGui:
 
     def __init__(self) -> None:
         self.thread: Optional[threading.Thread] = None
+        self.cancel_thread_event: threading.Event = threading.Event()
```

Comment:
Mypy knows it's an event:

```suggestion
        self.cancel_thread_event = threading.Event()
```